src/CommonDBTM.php

```diff
@@ -1445,7 +1445,7 @@ public function getLink($options = [])
 
         $link = $this->getLinkURL();
 
-        $label = $this->getNameID($options);
+        $label = htmlentities($this->getNameID($options));
```

Comment:
Since this is not changed a few lines above, method will retrun sanitized or not sanitized values. I'm not sure this is a good idea.

Also, I agree for changes in specific Search classes since this will impact nothing else ; but are we sure changes on commondbtm won't have any impact elsewhere?

Reply:
IMHO, methods that may outputs HTML should always return a safe html string, even if it does not contains any HTML tag. So, indeed, the returned value on line 1443 should also be sanitized.

Reply:
Well... I'd agree but I'm not sure this has never been the case in GLPI :/